test/units/plugins/lookup/test_laps_password.py

```diff
@@ -115,18 +115,17 @@ def test_missing_ldap(laps_password):
 
 
 def test_invalid_cert_mapping():
-    with pytest.raises(AnsibleLookupError) as err:
+    with pytest.raises(AnsibleError) as err:
         lookup_loader.get('laps_password').run(["host"], domain="test", validate_certs="incorrect")
 
-    assert str(err.value) == "Invalid validate_certs value 'incorrect': valid values are 'allow', 'demand', " \
-                             "'never', 'try'"
```

Comment:
Does it also make sense to remove the error checking code form lib/ansible/plugins/lookup/laps_password.py which produced the old messages?